repository-data/development/src/main/resources/hcm-actions.yaml

```diff
@@ -56,4 +56,9 @@ action-lists:
       /content/documents/lks/common: reload
       /content/documents/lks/guidance-pages: reload
   - 1.0.32:
-      /content/documents/lks/home: reload
\ No newline at end of file
+      /content/documents/lks/home: reload
+  - 1.0.33:
+      /content/documents/lks/home: reload
+      /content/documents/lks/common: reload
+      /content/documents/lks/guidance-pages: reload
+      /content/documents/lks/landing-pages: reload
```

Comment:
Think `/content/documents/lks/cookies` & `/content/documents/lks/listing-pages` also needs to be reloaded ?

Reply:
done